samples/client_query_w_positional_params.py

```diff
@@ -35,7 +35,9 @@ def client_query_w_positional_params() -> None:
             bigquery.ScalarQueryParameter(None, "INT64", 250),
         ]
     )
-    query_job = client.query(query, job_config=job_config)  # Make an API request.
+    results = client.query_and_wait(
+        query, job_config=job_config
+    )  # Make an API request.
 
     for row in query_job:
```

Comment:
This line needs to be updated too. The `query_job` variable doesn't exist anymore.

Reply:
Yes, will correct now.